.gitignore

```diff
@@ -1,2 +1,3 @@
 /.idea/
 .DS_Store
+/backend/src/main/resources/static/
```

Comment:
Manchmal möchte man tatsächlich auch statische Resourcen (Bilder, etc.) einchecken. Ich vermute, dass Du das Verzeichnis ausgenommen hat um das fertig kompilierte Frontend hier ablegen zu können.

Für den Moment stört das ignore ja nicht - aber falls Du mal etwas dort einchecken möchtest, müsstest Du eben dann zu gegebener Zeit hier differenzierter ignore-ren.

Reply:
Ja, deine Vermutung ist richtig. Nachdem ich das Frontend hinzugefügt hatte, hatte ich jede Menge redundante Daten in dem Verzeichnis, die ich nicht im Codereview oder auf Github haben wollte. Ich werde an Deine Anmerkung denken, wenn ich eigene statische Daten hinzufüge.